### DISCUSSION TITLE
Billwerk Payments integration

### DISCUSSION THREAD

**Discussion by vvarp** [juspay/hyperswitch#3199] [Ideas / Feature Requests]
Billwerk Payments (formerly Reepay) is a payment gateway provider focused on Northern Europe countries.

- Product: https://www.billwerk.plus/payments/
- Docs: https://docs.reepay.com/

**Comment by SanchithHegde** [comment]
Corresponding issue for this discussion: #3227

**Comment by SamraatBansal** [comment]
Hey @vvarp !

We are glad to inform that we have integrated [Billwerk](https://www.billwerk.plus/pay-payment-gateway/) with Hyperswitch, and is currently live in our Sandbox Dashboard from [2024.04.02.0](https://github.com/juspay/hyperswitch/commit/5587a126844abd580c67c8e2b05c26502cd6d844)+. You can try the same at [Hyperswitch Dashboard](https://app.hyperswitch.io).
